shared/minime/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@aragon-apps/minime",
```

Comment:
Note that I've opted to use the prefix `@aragon-apps` for private packages that should only be shared inside this repo.